cirq-core/cirq/json_resolver_cache.py

```diff
@@ -51,6 +51,21 @@ def two_qubit_matrix_gate(matrix):
             matrix = np.array(matrix, dtype=np.complex128)
         return cirq.MatrixGate(matrix, qid_shape=(2, 2))
 
+    def _cross_entropy_result(data, repetitions, **kwargs) -> Dict[str, Any]:
+        purity_data = kwargs.get('purity_data', None)
+        if purity_data is not None:
+            purity_data = [(d, f) for d, f in purity_data]
+        return {
```

Comment:
should this be a named tuple instead?  that wa they actually mimic the way they are used for the previous data class?  This is also works for the result dict.

Reply:
Ok, everything is deserialized into a NamedTuple now.  What do you think?